.nvmrc

```diff
@@ -0,0 +1 @@
+v18.15.0
```

Comment:
What do you think about adding an `.nvmrc` to this with the [same version](https://github.com/getflywheel/flywheel-local/blob/ab9a4a58947fe0a93813c7102b7278d4ec870326/.nvmrc) as Local Core?